packages/zosfiles/src/methods/list/doc/IDsmListOptions.ts

```diff
@@ -0,0 +1,41 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ITaskWithStatus } from "@zowe/imperative";
+import { IZosFilesOptions } from "../../../doc/IZosFilesOptions";
+
+/**
+ * This interface defines the options that can be sent into the list data sets matching function
+ */
+export interface IDsmListOptions extends IZosFilesOptions {
+    /**
+     * Exclude data sets that match these DSLEVEL patterns. Any data sets that match
+     * this pattern will not be listed
+     * @example "ibmuser.**.jcl, ibmuser.rexa.*"
+     */
+    excludePatterns?: string[];
```

Comment:
Should we deprecate the `excludePatterns` property on the `IDownloadOptions` interface? I don't think it is being used anywhere there, and having it there may be misleading since now we have it on this interface instead.